packages/lesswrong/components/posts/TableOfContents/MultiToCLayout.tsx

```diff
@@ -168,13 +168,15 @@ export type ToCLayoutSegment = {
   isCommentToC?: boolean,
 };
 
-const MultiToCLayout = ({segments, classes, tocRowMap = [], showSplashPageHeader = false, answerCount, commentCount}: {
+const MultiToCLayout = ({segments, classes, tocRowMap = [], showSplashPageHeader = false, answerCount, commentCount, tocContext}: {
   segments: ToCLayoutSegment[],
   classes: ClassesType<typeof styles>,
   tocRowMap?: number[], // This allows you to specify which row each ToC should be in, where maybe you want a ToC to span more than one row
   showSplashPageHeader?: boolean,
   answerCount?: number,
   commentCount?: number,
+  fixedTocCommentCountHeight?: number,
```

Comment:
This doesn't actually need to be a prop here, right

Reply:
Oops, I thought I removed that